grails-app/views/returns/show.gsp

```diff
@@ -130,7 +130,14 @@
                                 <g:message code="stockMovement.status.label"/>
                             </td>
                             <td class="value">
-                                <format:metadata obj="${stockMovement?.status ?: (stockMovement?.shipment?.status?.code ?: stockMovement?.statusCode)}"/>
+                                <g:set var="isInbound" value="${stockMovement?.destination == currentLocation}" />
```

Comment:
You could (should) use a `stockMovementDirection` here (but first look at the comment from the line below)

Reply:
I could for the `StockMovement`, but could not for the `OutboundStockMovement`, as this property/method is missing there.